Add Community Contributor 2019 newspost and assets (#3154)
ppy/osu-wiki: news/2020-02-04-community-contributors-2019.md

---
layout: post
title: "Community Contributors: 2019"
date: 2020-02-04 10:00:00 +0000
---

osu! thrives on the unsung efforts of its community, and among them, a few exemplars arise. These people selflessly dedicate themselves towards furthering the game that you know and love, and today, we take a moment to honour their contribution.

![](/wiki/shared/news/2020-02-04-community-contributors-2019/header.jpg)

Perhaps the most prestigious of all profile badges awarded, the Community Contributor badge represents enduring contribution to the development of the osu!community through a variety of disciplines and fields. 

<div align="center">
    <img src="https://assets.ppy.sh/profile-badges/contributor.jpg"/>
</div>

Those who receive it have undoubtedly left their mark on the game for the better in one way or another, and today, we grant several more.

## Community

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/vinxis.jpg)](https://osu.ppy.sh/users/4323406)

> **VINXIS** is chiefly responsible for the organization and upkeep of several major community events, such as the *Corsace Open*, the *OHC*, the *Mapper's Choice Awards* and has also extensively assisted in prospective trials towards improving the cornerstone of the game's competitive progression, the performance points system.

Reviewer: ```suggestion
**VINXIS** is chiefly responsible for the organization and upkeep of several major community events, such as *Corsace Open*, the *OHC*, the *Mapper's Choice Awards* and has also extensively assisted in prospective trials towards improving the cornerstone of the game's competitive progression, the performance points system.
```

An organizer of the Community Mentorship Program and a long-term liaison between the osu!team and the Global Moderation Team, **Chaos** has been a stabilizing force in the community for many years now, with his efforts at home and abroad solidifying the future of the game. Beyond just this, he has also been heading the osu! Mapping Olympiad, a premier beatmapping contest series all the while.

His work in managing the upkeep of the GMT and ensuring that the newest generations of osu!'s mappers and modders are guided through a labyrinthine set of customs and traditions, and helping them integrate into the game proper is essentially futureproofing the scene for years to come, whilst his efforts as a moderator ensure that the game's chat and forums remain safe for everyone.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/yaspo.jpg)](https://osu.ppy.sh/users/4945926)

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Uberzolik.jpg)](https://osu.ppy.sh/users/1314547)

Essential to the continuation of the Community Mentorship Program, **yaspo and Uberzolik** have spent thousands of hours performing key tasks to ensure things stay on track.

> After Halfslashed stepped down, **yaspo** picked up the slack and stepped up to lead beside Okoratu, and has been responsible for running dozens of contests, hundreds of discussions, hand-picking similar numbers of mentor and mentee applications, hosting meetings and so much more.

Reviewer: ```suggestion
After Halfslashed stepped down, **yaspo** picked up the slack and stepped up to lead beside Okoratu, and has been responsible for running dozens of contests and discussions, hand-picking several hundreds of mentor and mentee applications, hosting meetings and so much more.
```

Discussions aren't nearly that many, but mentor and mentee applicantions certainly are

A chief contributor to the Mappers' Guild and Beatmap Nominator infrastructure, **Milan-**'s advice and aid have helped shape the backbone of the systems that govern osu!'s ranking cycle, and one of its primary projects for promoting the use of officially licensed music, all while maintaining his place in the Beatmap Nominators.

## Nominators

In an effort to acknowledge the people who form the spine of osu!'s ranking system, we will be awarding a new form of Elite badges each year to commemorate the contributions of that year's most impressive Beatmap Nominators.

<div align="center">
    <img src="https://assets.ppy.sh/profile-badges/elite-nominator.png"/>
</div>

Only the most engaged qualify for such an accolade, those who truly stand at the top of their cohort as inarguably the most active and engaged Beatmap Nominators of all.

To acknowledge these efforts, the Nomination Assessment Team have come together to write the following accolades describing what each of the Elite Nominators have done.

### osu!

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Mordred.jpg)](https://osu.ppy.sh/users/7265097)

> **Mordred** is a stand out individual who has been extremely active in all of nominating, resetting, qualifying and disqualifying maps. On top of this, his enthusiasm towards pushing full version maps has not gone unnoticed. He definitely cares quite a lot about the quality of mapping on osu!, and his persistence goes to show how he deserves the title of Elite Nominator.

Reviewer: ```suggestion
**Mordred** is a stand out individual who has been extremely active in all of nominating, resetting, qualifying and disqualifying maps. On top of this, his enthusiasm towards pushing full version maps has not gone unnoticed. He definitely cares quite a lot about the quality of mapping on osu!, and his dedication goes to show how he deserves the title of Elite Nominator.
```
"Persistence" seems like an odd choice of words, given there's no opposition of sorts mentioned in the text.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Cheri.jpg)](https://osu.ppy.sh/users/5226970)

**Cheri** has been a supremely active nominator and very approachable for all kinds of issues. She does not hesitate to contribute wherever she sees possible, and this also shows in her hard work checking qualified as a QAH member and actively posting problems and suggestions on maps that can be improved further! She has been excelling as a Beatmap Nominator, and her kindness makes that even better. As someone doing so well, she embodies the kind of passion needed to be an Elite Nominator.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/eiri.jpg)](https://osu.ppy.sh/users/3388410)

> Everything you'd expect from a beatmap nominator and more, **eiri-** is definitely one of the best around. She's super active and excels in all of nominating, popping, and disqualifying maps, as well as generally checking maps in qualified. In addition to that, she's been very active in pushing maps from mappers both new and old across tons of musical genres.With everything covered, eiri- is definitely good at what she does!

Reviewer: ```suggestion
Everything you'd expect from a beatmap nominator and more, **eiri-** is definitely one of the best around. She's super active and excels in all of nominating, popping, and disqualifying maps, as well as generally checking maps in qualified. In addition to that, she's been very active in pushing maps from mappers both new and old across tons of musical genres. With everything covered, eiri- is definitely good at what she does!
```

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/bossandy.jpg)](https://osu.ppy.sh/users/360437)

> There are those who are incredible simply by being themselves. **bossandy** is not only a good nominator in terms of modding, but he is also very open minded about different mapping styles and always try to push maps from all kinds of difficulties regardless of how unconventional it may be and still maintain a good proficiency. His great performance paired with his very kind personality is what makes him stand out and earn not only the respect of others, but also the title of Elite Nominator!

Reviewer: ```suggestion
There are those who are incredible simply by being themselves. **bossandy** is not only a good nominator in terms of modding, but he is also very open minded about different mapping styles. He always tries to push maps from all kinds of difficulties, regardless of how unconventional it may be, while still maintain a good proficiency. His great performance paired with his very kind personality is what makes him stand out and earn not only the respect of others, but also the title of Elite Nominator!
```
The sentence in the middle is quite long, better to split it up. (By the way, not entirely sure what "proficiency" means in this context)

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Lasse.jpg)](https://osu.ppy.sh/users/896613)

> While also serving as a key member of the QAT and later the NAT, **Lasse** has managed to maintain really high activity in nominating as well. He successfully nominated over 100 maps this year, and also disqualified more than anyone else,  all while having a keen eye in his modding. Lasse's mods are some of those that consistently understand what mappers are doing, spotting every mistake, and more!

Reviewer: ```suggestion
While also serving as a key member of the QAT and later the NAT, **Lasse** has managed to maintain really high activity in nominating as well. He successfully nominated over 100 maps this year, and also disqualified more than anyone else, all while having a keen eye in his modding. Lasse's mods are some of those that consistently understand what mappers are doing, spotting every mistake, and more!
```

<div align="center">
    <img src="https://assets.ppy.sh/profile-badges/contributor.jpg"/>
</div>

Those who receive it have undoubtedly left their mark on the game for the better in one way or another, and today, we grant several more.

## Community

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/vinxis.jpg)](https://osu.ppy.sh/users/4323406)

**VINXIS** is chiefly responsible for the organization and upkeep of several major community events, such as *Corsace Open*, the *OHC*, the *Mapper's Choice Awards* and has also extensively assisted in prospective trials towards improving the cornerstone of the game's competitive progression, the performance points system.

Not content to leave it there, he is also a [Featured Artist](https://osu.ppy.sh/beatmaps/artists/22) and one of the original members of the Mappers' Guild. Few have made such enduring changes upon osu!'s landscape as VINXIS and his efforts have, and it is safe to say that without him, osu! would have been considerably worse off.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/hehe.jpg)](https://osu.ppy.sh/users/2123087)

> Previously known as *handsome*, **hehe** is a founding member of the Corsace project, an original Mappers' Guild founding member and most renowned not only for his many creative masterpieces in the mapping domain, but also for his visual design and organizational contributions to some of the game's most enduring community-run projects.

Reviewer 2: might be worth mentioning some of those community-run projects

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Noffy.jpg)](https://osu.ppy.sh/users/1541323)

A universal fixture within the NAT, **Noffy** has quickly made a name for herself as one of the prime contributors towards the continued maintenance and development of the game as far as the modding, mapping and metadata goes. 

Her continued oversight and assistance with GitHub issue tracking, contribution to the Mappers' Guild and her general all-around presence exemplify a spirit that has driven some of the osu! community's most notable people into shaping the game into what it is today.

We earnestly hope to see more of her ilk rise to the fore in the future, and we are immensely grateful for her contribution.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Shoegazer.jpg)](https://osu.ppy.sh/users/2520707)

> A true paragon-in-waiting, few are as synonymous with a game mode's excellence as **Shoegazer** is with osu!mania. His contributions towards the mode are vast beyond reckoning, ranging from groundwork in dealing with criteria and gameplay issues to the development of tournament-grade map pools for the game's finest players, featuring extensively in osu!mania's two most vaunted tournaments - the osu!mania World Cup and SOFT.

Reviewer 2: changing 'criteria' to 'the ranking criteria' would be clearer for most users

His work in managing the upkeep of the GMT and ensuring that the newest generations of osu!'s mappers and modders are guided through a labyrinthine set of customs and traditions, and helping them integrate into the game proper is essentially futureproofing the scene for years to come, whilst his efforts as a moderator ensure that the game's chat and forums remain safe for everyone.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/yaspo.jpg)](https://osu.ppy.sh/users/4945926)

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Uberzolik.jpg)](https://osu.ppy.sh/users/1314547)

Essential to the continuation of the Community Mentorship Program, **yaspo and Uberzolik** have spent thousands of hours performing key tasks to ensure things stay on track.

After Halfslashed stepped down, **yaspo** picked up the slack and stepped up to lead beside Okoratu, and has been responsible for running dozens of contests and discussions, hand-picking several hundreds of mentor and mentee applications, hosting meetings and so much more.

> **Uberzolik** has been responsible for maintaining huge swathes of data required for the project to continue, balancing this workload whilst also being a mentor AND a Beatmap Nominator himself.

Reviewer: I kinda wish this mentioned more than just the data stuff, since I essentially fill in the blanks for yaspo or directly help him with certain things. I mainly lead mentee checkups, which is meant to assure mentee-mentor interaction is working effectively/things are being done. Besides that, I handle mentor and mentee apps alongside yaspo (and a few others, sometimes). Dunno if it's wrong for me to bring this up on my own message for the newspost lol

Author: i've reworded your blurb to refer to the above

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Death.jpg)](https://osu.ppy.sh/users/3242450)

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Dntm8kmeeatu.jpg)](https://osu.ppy.sh/users/5428812)

Combined, **Death and Dntm8kmeeatu** put the support in the Support Team, collectively responsible for solving literally thousands of problems and answering questions for those in need, all while balancing their duties as members of the Global Moderation Team.

## Development

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Milan.jpg)](https://osu.ppy.sh/users/1052994)

A chief contributor to the Mappers' Guild and Beatmap Nominator infrastructure, **Milan-**'s advice and aid have helped shape the backbone of the systems that govern osu!'s ranking cycle, and one of its primary projects for promoting the use of officially licensed music, all while maintaining his place in the Beatmap Nominators.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Joehu.jpg)](https://osu.ppy.sh/users/8549835)

> **Joehu** has been a part of the osu!wiki team and a regular contributor to the osu!lazer project for many years now, and has rapidly made a name for himself as one of the most fastidious and efficient reviewers overall. Wikifriends near and far regularly comment that if you get a green light from Joehu, you know you're set, and when folklore legends are arising out of someone's attention to detail, there's really not much story to tell at that point.

Reviewer 2: > there's really not much story to tell at that point.

unsure if this is just me being a foreigner but the wording here sounds wrong to me

Reviewer: > "not much story left to tell"

 might be clearer

Reviewer 2: even that sounds confusing. i think changing the sentence completely would be a better starting point

> ## Nominators

Reviewer 3: I get mixed feelings about this cuz there's already rewards of the same type for dedicated nominators, and according to these descriptions, the basis for "elite nominator" seems to be more=better for most of the people. I think the subjective nature of what makes someone a "good" nominator will be problematic for how these badges are viewed. I feel like this is the kind of thing that should have been discussed beyond just the NAT beforehand, I doubt I'm the only person with this concern and now it's too late to make changes (I assume)

to be clear I'm mostly complaining about how this is a whole separate badge with different implications than "community contributor", which would imo be more fitting.

Author: they're chosen based on a review of metrics + performance as discussed with the NAT, so they're essentially contributor-like badges that have a baseline requirement for consideration

Reviewer 3: which is why I feel like community contributor would've been more appropriate. "elite nominator" by itself gives some impression that they do *better* work than normal BNs, something you could easily disagree with if your mapping/modding opinions don't align

<div align="center">
    <img src="https://assets.ppy.sh/profile-badges/elite-nominator.png"/>
</div>

Only the most engaged qualify for such an accolade, those who truly stand at the top of their cohort as inarguably the most active and engaged Beatmap Nominators of all.

To acknowledge these efforts, the Nomination Assessment Team have come together to write the following accolades describing what each of the Elite Nominators have done.

### osu!

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Mordred.jpg)](https://osu.ppy.sh/users/7265097)

**Mordred** is a stand out individual who has been extremely active in all of nominating, resetting, qualifying and disqualifying maps. On top of this, his enthusiasm towards pushing full version maps has not gone unnoticed. He definitely cares quite a lot about the quality of mapping on osu!, and his dedication goes to show how he deserves the title of Elite Nominator.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Cheri.jpg)](https://osu.ppy.sh/users/5226970)

> **Cheri** has been a supremely active nominator and very approachable for all kinds of issues. She does not hesitate to contribute wherever she sees possible, and this also shows in her hard work checking qualified as a QAH member and actively posting problems and suggestions on maps that can be improved further! She has been excelling as a Beatmap Nominator, and her kindness makes that even better. As someone doing so well, she embodies the kind of passion needed to be an Elite Nominator.

Reviewer 2: changing 'qualified' to 'the Qualified section' would be clearer to most users

> **Cheri** has been a supremely active nominator and very approachable for all kinds of issues. She does not hesitate to contribute wherever she sees possible, and this also shows in her hard work checking qualified as a QAH member and actively posting problems and suggestions on maps that can be improved further! She has been excelling as a Beatmap Nominator, and her kindness makes that even better. As someone doing so well, she embodies the kind of passion needed to be an Elite Nominator.

Reviewer 2: > She has been excelling as a Beatmap Nominator, and her kindness makes that even better

would probably word that a bit differently to reflect her kindness better. maybe something like

> She has been excelling as a Beatmap Nominator, and her overwhelming kindness only helps to further cement that work.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/eiri.jpg)](https://osu.ppy.sh/users/3388410)

> Everything you'd expect from a beatmap nominator and more, **eiri-** is definitely one of the best around. She's super active and excels in all of nominating, popping, and disqualifying maps, as well as generally checking maps in qualified. In addition to that, she's been very active in pushing maps from mappers both new and old across tons of musical genres. With everything covered, eiri- is definitely good at what she does!

Reviewer 2: changing 'qualified' to 'the Qualified section' would be clearer to most users

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/bossandy.jpg)](https://osu.ppy.sh/users/360437)

There are those who are incredible simply by being themselves. **bossandy** is not only a good nominator in terms of modding, but he is also very open minded about different mapping styles. He always tries to push maps from all kinds of difficulties, regardless of how unconventional it may be, while still maintain a good proficiency. His great performance paired with his very kind personality is what makes him stand out and earn not only the respect of others, but also the title of Elite Nominator!

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Lasse.jpg)](https://osu.ppy.sh/users/896613)

> While also serving as a key member of the QAT and later the NAT, **Lasse** has managed to maintain really high activity in nominating as well. He successfully nominated over 100 maps this year, and also disqualified more than anyone else, all while having a keen eye in his modding. Lasse's mods are some of those that consistently understand what mappers are doing, spotting every mistake, and more!

Reviewer 2: > While also serving as a key member of the QAT and later the NAT, **Lasse** has managed to maintain really high activity in nominating as well.

Sentence sounds overly complicated. Simplifying it a bit could look like 

> While also serving as a key member of the QAT and later the NAT, **Lasse** has managed to maintain extraordinarily high nomination activity.

### osu!taiko

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/komasy.jpg)](https://osu.ppy.sh/users/1980256)

> **komasy** made a huge impact on osu!taiko during 2019. Within his 9 month tenure before resigning, komasy helped to push over **100** mapsets to the ranked category! On top of that, he has been very easy to approach, always responding if you have a question or need an opinion. komasy has helped osu!taiko as a mode, helped newer mappers to figure things out, and been a beacon of knowledge in the community.

Reviewer 2: > komasy helped to push over **100** mapsets to the ranked category!

Sounds a bit off. maybe this would be better

> komasy helped push over **100** mapsets to the ranked category!

> **komasy** made a huge impact on osu!taiko during 2019. Within his 9 month tenure before resigning, komasy helped to push over **100** mapsets to the ranked category! On top of that, he has been very easy to approach, always responding if you have a question or need an opinion. komasy has helped osu!taiko as a mode, helped newer mappers to figure things out, and been a beacon of knowledge in the community.

Reviewer 2: > helped newer mappers to figure things out

fillerword

> helped newer mappers figure things out

> **komasy** made a huge impact on osu!taiko during 2019. Within his 9 month tenure before resigning, komasy helped to push over **100** mapsets to the ranked category! On top of that, he has been very easy to approach, always responding if you have a question or need an opinion. komasy has helped osu!taiko as a mode, helped newer mappers to figure things out, and been a beacon of knowledge in the community.

Reviewer 2: > On top of that, he has been very easy to approach, always responding if you have a question or need an opinion.

could be simplified further

> On top of that, he has been very approachable, always responding to questions or people in need of opinions.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Tyistiana.jpg)](https://osu.ppy.sh/users/1421452)

> Having stable activity throughout 2019, **Tyistiana** takes care with each and every mapset. Additionally, he actively partakes in disqualification discussions, and he recently does the vast majority of QAH checks too. He is serving actively both to push content and assure the quality of maps going for ranked, and is truly a backbone for the osu!taiko game mode!

Reviewer 2: Imo. this whole paragraph needs some restructuring and simplification to correctly reflect the tone of everything else. Here's my take:

> **Tyistiana** has through stable activity and careful work as a Beatmap Nominator helped further osu!taiko as a game mode. Additionaly, he has actively partaken in disqualification discussions and done work inside the qualified section to assure a high quality of ranked maps. Tyistiana is one who can truly be called a backbone of osu!taiko.

Reviewer 2: also, mentioning the specific role he has done instead of relying on the user to _know_  beforehand would be good.

edit: i changed the above comment to reflect this change

### osu!catch

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Jemzuu.jpg)](https://osu.ppy.sh/users/7890134)

> After a bit of a rocky start, osu!catch's longest standing nominator in terms of consecutive time (430 days!) has been nothing less than extraordinary over the last year. Boasting nearly unmatched statistics in resets, disqualifications, and nominations on top of an incredibly impressive tenure in the Quality Assurance Helpers, **Jemzuu** has been a massive help in just about every field that osu!catch has to offer. When it comes to being an Elite Nominator, it would be nearly impossible to overlook him.

Reviewer 2: Should use Beatmap Nominator to stay consistent across the article

> After a bit of a rocky start, osu!catch's longest standing nominator in terms of consecutive time (430 days!) has been nothing less than extraordinary over the last year. Boasting nearly unmatched statistics in resets, disqualifications, and nominations on top of an incredibly impressive tenure in the Quality Assurance Helpers, **Jemzuu** has been a massive help in just about every field that osu!catch has to offer. When it comes to being an Elite Nominator, it would be nearly impossible to overlook him.

Reviewer 2: > has been nothing less than extraordinary over the last year.

could be simplified

> has been nothing less than extraordinary in/across/throughout 2019.

> After a bit of a rocky start, osu!catch's longest standing nominator in terms of consecutive time (430 days!) has been nothing less than extraordinary over the last year. Boasting nearly unmatched statistics in resets, disqualifications, and nominations on top of an incredibly impressive tenure in the Quality Assurance Helpers, **Jemzuu** has been a massive help in just about every field that osu!catch has to offer. When it comes to being an Elite Nominator, it would be nearly impossible to overlook him.

Reviewer 2: > When it comes to being an Elite Nominator, it would be nearly impossible to overlook him.

could add a bit to this, and make a bigger impact of his work

> When it comes to being an Elite Nominator, it is impossible to overlook Jemzuu's incredible work and contribution to osu!catch.

> After a bit of a rocky start, osu!catch's longest standing nominator in terms of consecutive time (430 days!) has been nothing less than extraordinary over the last year. Boasting nearly unmatched statistics in resets, disqualifications, and nominations on top of an incredibly impressive tenure in the Quality Assurance Helpers, **Jemzuu** has been a massive help in just about every field that osu!catch has to offer. When it comes to being an Elite Nominator, it would be nearly impossible to overlook him.

Reviewer 2: I think it'd be better to mention _who_ is receiving the reward at the start of the paragraph

> After a bit of a rocky start, osu!catch's longest standing nominator in terms of consecutive time (430 days!) has been nothing less than extraordinary over the last year. Boasting nearly unmatched statistics in resets, disqualifications, and nominations on top of an incredibly impressive tenure in the Quality Assurance Helpers, **Jemzuu** has been a massive help in just about every field that osu!catch has to offer. When it comes to being an Elite Nominator, it would be nearly impossible to overlook him.

Reviewer 2: >Boasting nearly unmatched statistics in resets, disqualifications, and nominations on top of an incredibly impressive tenure in the Quality Assurance Helpers, **Jemzuu** has been a massive help in just about every field that osu!catch has to offer.

this sentence is a bit long. consider splitting it up

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Kobii.jpg)](https://osu.ppy.sh/users/6209713)

One of the most active osu!mania nominators since joining in early 2019, **\_Kobii** has been active in pushing maps in not just the 4k mode, or the 7k mode, but also 6k, 9k, and more! On top of that, they have been very approachable for mappers new and old in the osu!mania mapping community and serve as a great example of what a Beatmap Nominator can be!

-------

2019 was one of the strongest years osu! has had so far, and it is in no small part due to the efforts of those listed above.

We know that so many people contribute to help make osu! awesome, and those we have awarded today are by no means the sole, exhaustive list of everyone who deserves one.

> osu! is unique among the thousands of communities on the internet in that we are shaped so very closely by those among us who choose to help, and we are all privileged to share in the result of so many people working together for a common goal.

Reviewer 2: unsure, but shouldnt this be 'amongst' ?

Author: US vs UK english, amongst is preferred in UK, among in US

**komasy** made a huge impact on osu!taiko during 2019. Within his 9 month tenure before resigning, komasy helped push over **100** mapsets to the ranked category! On top of that, his approachability and willingness to help have cultivated a glowing reputation among the scene. komasy has helped build osu!taiko as a gamemode and a community, never shying away from assisting newer mappers with their work, and has generally been a beacon of knowledge all around.

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Tyistiana.jpg)](https://osu.ppy.sh/users/1421452)

Showcasing stability and consistency throughout 2019, **Tyistiana** has been an active nominator on a mission, dominating the performance charts among the Quality Assurance Helpers in his tireless quest to both help mappers push their work into Ranked, and assure their quality at the same time. His efforts have been instrumental towards the health of the osu!taiko scene during the year.

### osu!catch

[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Jemzuu.jpg)](https://osu.ppy.sh/users/7890134)

> After a bit of a rocky start, **Jemzuu** is osu!catch's longest standing beatmap nominator in terms of consecutive time (430 days!) has been nothing less than extraordinary throughout 2019. Boasting nearly unmatched statistics in resets, disqualifications, and nominations on top of an incredibly impressive tenure in the Quality Assurance Helpers, he has truly demonstrated a desire and willingness to help. His incredible work ethic and contributions make him a prime and well-deserved choice for an Elite Nominator.

Reviewer 2: needs an 'and' after '(430 days!)'
